README.md

```diff
@@ -3,6 +3,20 @@
 This is a repository for the AI Engineering team to share code with customers and the community.  This ranges from simple example API snippets to demonstrations and end to end tutorials and associated code.
 
 
+Additional code examples can be accessed from the following locations:
+
+- [Tutorials for data scientists repo](https://github.com/datarobot-community/tutorials-for-data-scientists/)
+- [Examples for data scientists repo](https://github.com/datarobot-community/examples-for-data-scientists/)
+- [AI Engineering repo](https://github.com/datarobot-community/ai_engineering)
+- [docs.datarobot.com](https://docs.datarobot.com/en/docs/api/code-examples/index.html)
+- [R Vignettes included in the R client](https://cran.r-project.org/web/packages/datarobot/index.html)
+- [Examples included in the Python client](https://datarobot-public-api-client.readthedocs-hosted.com/en/v2.27.1/examples/index.html)
+
+## Important Links
+
```

Comment:
can you replace this content with the same changes you made for the other two repos:
```
- To learn how to use DataRobot and gain practical data science education, visit [DataRobot University](https://university.datarobot.com/)
- To ask questions and get help from DataRobot experts and peers, visit [DataRobot Community](https://community.datarobot.com/)
```